.github/workflows/ci.yml

```
@@ -13,6 +13,8 @@ jobs:
       - uses: actions/checkout@v2
 
       - uses: purescript-contrib/setup-purescript@main
+        with:
+          purescript: "unstable"
 
       - uses: actions/setup-node@v1
         with:
```

Oops, I didn't notice this. This should be:

```suggestion
      - uses: actions/setup-node@v2
        with:
          node-version: "14"
```

This will have to be added to the script.

---

I think this should stay on 12 because that IS the minimum version `purs` supports. If you want to also test 14, then we should do a matrix.